packages/ui-components/src/components/Chart/Table/useChartDataExport.js

```diff
@@ -3,8 +3,8 @@
  *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
 
-import { useDataTableExport } from '@tupaia/ui-components';
 import { getChartTableData } from './getChartTableData';
+import { useDataTableExport } from '../../DataTable/useDataTableExport';
```

Comment:
@tcaiger this was throwing an error, is there any issue with importing the file directly?

Reply:
Yes, we can't import the file directly because the Chart (and Map) directory gets bundled separately to the rest of the ui-components. It's definitely a trip hazard the way it's set up at the moment.